### ISSUE TITLE
Unable to connect to BIRDv4 socket: dial unix /var/run/calico/bird.ctl: connect: connection refused

### ISSUE THREAD

**Issue by hextrim** [projectcalico/calico#4834]
I have exactly the same problem. This is a wired issue.
I added node to a k8s cluster via kubespray.
It was working fine.
I shutdown cluster for the night and started in the morning the node I added to the cluster via kubespray has exactly the same problem as explained above.

Further:
[root@ht-prov-01 ~]# k logs -f pods/calico-node-2f497 -n kube-system
Error from server: Get "https://192.168.1.67:10250/containerLogs/kube-system/calico-node-2f497/calico-node?follow=true": dial tcp 192.168.1.67:10250: connect: no route to host

Basically calico is not operational on the node anymore.

Any idea how could this happen?

_Originally posted by @hextrim in https://github.com/projectcalico/calico/issues/4603#issuecomment-897133023_

**Comment by hextrim** [comment]
Ignore me - schoolboy error - new node has also firewalld enabled preventing calico ports and k8s components port comms.
